Example/NYTPhotoViewer-SwiftTests/NYTPhotosOverlayViewTests.swift

```diff
@@ -20,21 +20,21 @@ class NYTPhotosOverlayViewTests: XCTestCase {
         let overlayView = NYTPhotosOverlayView()
         let leftBarButtonItem = UIBarButtonItem(barButtonSystemItem: UIBarButtonSystemItem.Done, target: nil, action: nil)
         overlayView.leftBarButtonItem = leftBarButtonItem
-        XCTAssert(leftBarButtonItem == overlayView.navigationBar.items.first?.leftBarButtonItem)
+        XCTAssert(leftBarButtonItem == overlayView.navigationBar.items!.first?.leftBarButtonItem)
```

Comment:
should we force unwrap here?  Will this crash the tests if `items` is nil?  I am thinking maybe we can change the `!` to a `?` so the test would just fail instead of crash if `items` is nil.

FYI this swift business is new to me so let me know if I am reading this wrong or making incorrect assumptions.